kaggle/api/kaggle_api_extended.py

```diff
@@ -729,17 +744,29 @@ def competitions_list(self,
             category: category to filter result to
             group: group to filter result to
         """
-    if group and group not in self.valid_competition_groups:
-      raise ValueError('Invalid group specified. Valid options are ' +
-                       str(self.valid_competition_groups))
+    if group:
+      if group not in self.valid_competition_groups:
+        raise ValueError('Invalid group specified. Valid options are ' +
+                         str(self.valid_competition_groups))
+      if group == 'all':
+        group = CompetitionListTab.COMPETITION_LIST_TAB_DEFAULT
+      else:
+        group = self.lookup_enum(CompetitionListTab, group)
+    else:
+      # Breaking change: default to list all competitions, not just active ones.

```

Comment:
Why do we need to have a breaking change?  I see that the linked issue is about this, but it seems like it would be ok to be an option rather than the default.

Reply:
Yes, I'm on the fence about this.
`kaggle c list --category all`
would do it. I'll change to that.